snapcraft/internal/pluginhandler/_runner.py

```diff
@@ -116,6 +116,20 @@ def prime(self) -> None:
                 'override-prime', self._override_prime_scriptlet,
                 self._primedir)
 
+    def _get_env(self):
```

Comment:
I don't see a reason this should be a method of this class, I suggest we make it standalone.